apps/docs/app/page.tsx

```diff
@@ -1,9 +1,217 @@
+"use client";
+
+import { Button as HopperButton, HopperProvider } from "@hopper-ui/components";
+import Image from "next/image";
+import AccessibleIcon from "./home/assets/accessible.svg";
+import InternationalIcon from "./home/assets/international.svg";
+import TypescriptIcon from "./home/assets/typescript.svg";
+import DarkModeIcon from "./home/assets/dark-mode.svg";
+import FontSizeIcon from "./home/assets/font-size.svg";
+import LineHeightIcon from "./home/assets/line-height.svg";
+import MarginIcon from "./home/assets/margin.svg";
+import ArrowIcon from "./home/assets/arrow.svg";
+import SelectArrowIcon from "./home/assets/select-arrow.svg";
+import { type ColorScheme, ThemeContext } from "@/context/theme/ThemeProvider.tsx";
+import { useContext } from "react";
+import Button from "../components/button/Button";
+import { ExternalLinkIcon, Icon } from "@/components/icon";
+
+// eslint-disable-next-line max-len
+import { CalendarIcon, ChartBarIcon, CheckmarkIcon, DeleteIcon, EyeVisibleIcon, FilterIcon, FocusIcon, ItalicIcon, LightbulbIcon, LockIcon, MailIcon, NotebookIcon, PinIcon, ProfileIcon, ReactionIcon, RecurringIcon, RewindIcon, SearchIcon, ShareIcon, StarIcon, StickyIcon, SyncIcon, TeamIcon, ThumbsUpIcon, UnarchiveIcon, WarningIcon } from "@hopper-ui/icons";
+
+import "@hopper-ui/tokens/fonts.css";
+import "./home/home.css";
+
 export default function Home() {
+    const { colorMode } = useContext(ThemeContext);
+
+    const theme = colorMode as ColorScheme;
+
     return (
-        <div className="hd-wrapper hd-flex">
+        <div className="hd-wrapper">
             <main className="hd-home">
-                <h1 className="hd-display">Welcome to Workleap's <strong className="hd-text--strong">Hopper</strong> Design System.</h1>
-                <p className="hd-display__subtitle">The documentation is currently in <strong className="hd-text--strong">beta</strong>.</p>
+                <div className="hd-home__heading">
+                    <h1 className="hd-home__header">Leap into creativity</h1>
+                    <p className="hd-home__copy">Explore our Design System, where icons, tokens, and components are handpicked for ultimate simplicity and accessibility.</p>
```

Comment:
The class name seems a bit odd to me; it could be` hd-home__subtitle` or `hd-home__tagline`.